crates/turborepo-auth/src/lib.rs

```diff
@@ -11,3 +11,13 @@ mod ui;
 pub use auth::*;
 pub use error::Error;
 pub use server::*;
+
+/// Token is the result of a successful login. It contains the token string and
+/// a boolean indicating whether the token already existed on the filesystem.
+pub struct Token {
```

Comment:
Seems like we have multiple structs named token? At least according to the error above. Maybe we should rename one of them to be more specific

Reply:
Nit, but I feel like this would be a little more idiomatic as an enum:

```
enum Token {
  Existing(String),
  New(String)
}

impl Token {
  fn into_inner(self) -> String {
    match self {
      Token::Existing(s) => s,
      Token::New(s) => s
    }
  }
}
```

Reply:
Yeah I was in the process of moving the token out of the `login` and into `lib`, so the duplication came from upstream since I needed to do it earlier.

As for the `Enum`, I'm not opposed to doing something like
```
enum Token {
  Existing(String),
  New(String)
}
impl Token {
  fn into_inner(self) -> Token {
    match self {
      Self::Existing(s) => Token { /* filled in fields */ },
      Self::New(s) => Token { /* fill in fields */ }
    }
  }
}
```